plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererUserModificationTracker.kt

```diff
@@ -210,7 +210,7 @@ class CodeWhispererUserModificationTracker(private val project: Project) : Dispo
             codewhispererSuggestionIndex = suggestion.index,
             codewhispererTriggerType = suggestion.triggerType,
             credentialStartUrl = startUrl,
-            codewhispererUserGroup = CodeWhispererUserGroupSettings.getInstance().getUserGroup().name,
+            codewhispererUserGroup = CodeWhispererUserGroup.Control.name,
```

Comment:
do we need to keep the `CodeWhispererUserGroup.Control` or we can get rid of it entirely?

Reply:
yes, will remove it when https://github.com/aws/aws-toolkit-common/pull/858 is merged